packages/issuer-api/src/pods/certification-request/handlers/create-certification-request.handler.ts

```diff
@@ -36,6 +42,12 @@ export class CreateCertificationRequestHandler
         files,
         isPrivate
     }: CreateCertificationRequestCommand): Promise<CertificationRequestDTO> {
+        if (!isValidEthereumAddress(to)) {
```

Comment:
Please use the latest `ethers.utils.isAddress()`from the `ethers` package instead of a custom function, to avoid errors.

Reply:
Hmm ethers seems to return `false` for a non-checksummed address...

Reply:
No need for this, so long as it passes the tests.